app/views/validations/_demo.html.erb

```diff
@@ -2,21 +2,21 @@
   <div class="col-6 form-group <%= validation_status @user, :first_name %>">
     <%= f.label :first_name, "First Name" %>
     <%= f.text_field :first_name, class: "form-control form-control-lg",
-      data: { target: "validations.firstName", action: "input->validations#validate", reflex_permanent: true } %>
+      data: { reflex: "debounced:input", reflex_permanent: true } %>
```

Comment:
Is this too magic? Maybe it would be better to be a bit more explicit:

`data: { reflex: "debounced:input->validations#default_reflex}` ?

Reply:
I like the magic actually.